lib/util/git.js

```diff
@@ -72,6 +129,38 @@ exports.clone = function(repoUrl, repoPath) {
   }).return();
 };
 
+exports.commit = function(repoPath, files, message) {
```

Comment:
After this PR, I want to explore plugin APIs for welder. I don't think it needs to directly implement git management. This could let us have our git-executable plugin and a node-git plugin if node-git is still too experiment. (Not saying node-git is too experimental but users could make the choice or something.) Plugins could also mean support for other VCS (hg, svn, etc) and hosts (bitbucket, gitlab, etc). I'm hoping to expose an interface for other functionality being loosely coupled but I should put that in an issue instead of this inline code comment.